mixer/adapter/kubernetesenv/cache_test.go

```diff
@@ -63,3 +63,57 @@ func TestClusterInfoCache_Pod(t *testing.T) {
 		})
 	}
 }
+
+func TestClusterInfoCache_Workload_ReplicationController(t *testing.T) {
```

Comment:
This is fine.  Did you consider adding a ReplicationController test to `TestKubegen_Generate` ?

Reply:
Oh, let me add it, I didn't see that test.
Thanks.

Reply:
TestKubegen_Generate is also in the release-1.0 branch ?
I've prepared two PR, once for 1.0.x, and this one for master.

Reply:
It should be. I think I missed this on the 1.0.x branch PR.  It isn't critical, but that is where we've been hanging the tests that cover new workload types.

Reply:
@douglas-reid I've updated the TestKubegen_Generate test on this PR and in 1.0.x prepared here #9061 

There are still some issues but I think are unrelated to the patch.